tools/serverpod_cli/bin/generator/pgsql_generator.dart

```diff
@@ -34,19 +34,27 @@ class PgsqlGenerator {
     out += '--\n';
     out += '\n';
 
+    // Drop Table If Already Exist
```

Comment:
Why was this commented out?

Reply:
i want to implement this function. 
while development. i want to change table column & its properties. 
i'm looking for sql script to alter existing column.
i looked for script. but didn't get it.
tring to take a copy of old existing table and drop it and recreate it and then copy old data to newly created table and then to drop copy table.

still didn't know the script to copy existing data to new data which match column name without losing it.